_events/2024-0119-dev-triage-ml-commons.markdown

```diff
@@ -0,0 +1,38 @@
+---
+calendar_date: '2024-01-19'
+eventdate: 2024-01-19 01:30:00 -0700
```

Comment:
shows this at 1AM - if it's in the afternoon, should be

eventdate: 2024-01-19 13:30:00 -0700